### ISSUE TITLE
IOException: Unable to parse line as [TOKEN]

### ISSUE THREAD

**Issue by pvcastro** [inception-project/inception#2435]
**Describe the bug**
Hi there! I'm trying to parse an exported document from another annotation tool into Inception. I'm working with relation and NER annotations. I'm attaching here my settings for both layers.
I have generated the attached .tsv file and I'm getting the following error when I upload it to the project:

`Error while uploading document a.tsv: IOException: Unable to parse line as [TOKEN]: [6-3 155-163 LINDEISA *[4] PESSOA[4] FUNCAO 6-1[0-4] ]`

**To Reproduce**

Upload document to the project using `WebAnno TSV v3.2 (WebAnno v3.x)`. 

I'm attaching the json exported settings for both layers. Please let me know if any other settings are required.

**Expected behavior**
Expected the document to be properly uploaded with the existing annotations in the tsv file. 

**Screenshots**
![error](https://user-images.githubusercontent.com/12713359/124187936-ec55e800-da94-11eb-9492-b3fcea54bcb7.png)

![relation_layer_settings](https://user-images.githubusercontent.com/12713359/124187636-7d788f00-da94-11eb-969f-cecc0204d542.png)

**Please complete the following information:**
 - Version and build ID: 0.19.7 (2021-06-19 00:45:41, build 53edf712)
 - OS: Linux
 - Browser: chrome

**Additional context**
Sample tsv, NER json layer and Relation json layer:
[a.tsvt](https://github.com/inception-project/inception/files/6750704/a_tsv.txt)
[ner_layer.json](https://github.com/inception-project/inception/files/6750705/ner_layer_json.txt)
[relation_layer.json](https://github.com/inception-project/inception/files/6750706/relation_layer_json.txt)

I'm thinking that it might have something to do with tokenization, but since I'm uploading the tokenized tsv file, this shouldn't be an issue, right?

Thanks!

**Comment by reckart** [comment]
Thanks for the report. Without having looked at the data yet: can you try exporting the entire original project from WebAnno and then importing the entire project into INCEpTION?

**Comment by pvcastro** [comment]
Hi @reckart ! I'm actually not trying to import it from WebAnno. I'm exporting it from tagtog, another annotation tool. I've written a parser to generate the proper tsv file. I'm comparing the resulting tsv file from other annotations I had made in WebAnno, and they look good to me, even though they are not the **exact** same project. Do you want me to export the entire current inception project? I have other annotated documents on it I uploaded with CONLL-2003 format, with no relations yet.

**Comment by reckart** [comment]
Ok, so the problem is that you define invalid disambiguation IDs e.g. here:

```
6-3	155-163	LINDEISA	*[4]	PESSOA[4]	FUNCAO	6-1[0-4]
```

The numbers in the square brackets are [disambiguation IDs](https://inception-project.github.io/releases/0.19.7/docs/user-guide.html#_disambiguation_ids). Such IDs are typically simple numbers. For relations they can be two numbers separated by an underscore `_` to indicate the disambiguation IDs for the relation source/target spans.

**Comment by pvcastro** [comment]
Thanks, @reckart , I didn't notice this :man_facepalming: ! Now I managed to import with no issues.

**Comment by jcklie** [comment]
Is it possible that you share your script so we can make it public to other people?

**Comment by reckart** [comment]
De Nada 😀

**Comment by pvcastro** [comment]
> Is it possible that you share your script so we can make it public to other people?

Sure, I wouldn't mind at all @jcklie . There are just a few issues, if you think it's ok:
1. It's very tightened to converting from tagtog to webanno/inception.
2. It's a python script that it's not very optimized :sweat_smile: . I had several issues trying to work with tagtog's metadata since they don't keep anything but characters indices. I originally was working with webanno and then tried evaluating tagtog, so I wrote a script to convert my annotations to it. I had to come up with some ways to parse and tokenize text from webanno to tagtog and backward, in a way that I would keep all characters coordinates exactly the same, while also trying to split the sentences in a cohesive way, to make sure I wouldn't lose any entities annotations. 
3. It's a bit coupled to my data, so I would have to do some cleaning/decoupling first.
